ddtrace/opentracer/span.py

```diff
@@ -147,12 +116,18 @@ def set_tag(self, key, value):
 
         This sets the tag on the underlying datadog span.
         """
-        return self._dd_span.set_tag(key, value)
+        if key == OTTags.SPAN_TYPE:
+            self._dd_span.span_type = value
+        elif key == OTTags.HTTP_URL or key == OTTags.DB_STATEMENT:
```

Comment:
`OTTags.HTTP_URL` introduces a high cardinality issue, so stats will not be useful for end users. If we set the SQL query (`OTTags.DB_STATEMENT`) as resource (and only as resource), it will pass our quantizer and obfuscator (security reasons) that will make stats useful.